src/modules/transaction/components/TransactionInfo/TransactionInfo.js

```diff
@@ -71,7 +75,13 @@ const TransactionInfo = ({
               <label>{date || '-'}</label>
             </div>
             <div className={styles.col}>
-              <label>{t('Nonce')}</label>
+              <label>
+                {t('Nonce')}{' '}
+                {nonceWarning && <Icon name="warningYellow" className={styles.warning} />}
+                {canResetNonce && (
```

Comment:
Why this new var. If we have a nonceWarning we should be able to reset 🤔

Reply:
Nonce should should only be reset if transaction hasn't been signed yet. That's why another variable was needed